.travis.yml

```diff
@@ -69,9 +69,9 @@ script:
   - if [ "$KNAPSACK_PRO_CI_NODE_INDEX" == "1" ]; then bundle exec bundle-audit check --update --ignore CVE-2015-9284; fi
   - if [ "$KNAPSACK_PRO_CI_NODE_INDEX" == "1" ]; then yarn build-storybook; fi
   - if [ "$KNAPSACK_PRO_CI_NODE_INDEX" == "2" ]; then bin/test-console-check; fi
-  # - yarn e2e:ci
-  # Dropping and recreating the database for Capybara/rspecs
   - export FOREM_OWNER_SECRET="secret"
+  - yarn e2e:ci
```

Comment:
I put this after the `FOREM_OWNER_SECRET` is set because it's required for the tests to run properly.